orderreturn.go

```diff
@@ -2,6 +2,12 @@ package stripe
 
 import "encoding/json"
 
+// OrderReturnResourceParams is the set of parameters that can be used when retrieving a transaction.
+// It is not to be confused with OrderReturnParams, which is used to return an order.
```

Comment:
You should use `OrderReturnParams` instead of making a new one, like we do in other clients I think